src/main/java/de/zalando/zmon/dataservice/config/DataServiceConfigProperties.java

```diff
@@ -17,6 +17,11 @@
     private int redisPort = 6378;
     private int redisPoolSize = 20;
 
+    // Datapoints Redis buffer for KairosDB
```

Comment:
I would also mention here that "datapoints Redis" means "Redis queue for sending datapoints to Kairos". People may initially think that this configuration sets up Redis as a data storage, not as a queue

Reply:
OK